src/coreclr/src/gc/gcwks.cpp

```diff
@@ -21,6 +21,10 @@
 #undef SERVER_GC
 #endif
 
+#if defined(TARGET_AMD64)
```

Comment:
I suppose you might as well also change this to check for `TARGET_WINDOWS` to be consistent.

Reply:
Yep - thanks!